docs/couchdb.md

```diff
@@ -0,0 +1,20 @@
+# Couchdb
+
```

Comment:
Note for Jimmy, the full documentation can be found in https://github.com/GoogleCloudPlatform/ops-agent/blob/90d604b692cffb9cf388948e586cf5e2a894b2d0/docs/couchdb.md

Note for future reference: Currently we are trying to automate the documentation generation for https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party. Before that is completed, the workflow is:
1. ObserveIQ contributes documentation (in the format of https://github.com/GoogleCloudPlatform/ops-agent/blob/90d604b692cffb9cf388948e586cf5e2a894b2d0/docs/couchdb.md) as `docs/{APP_NAME}.md` files.
2. Jimmy to take these markdown and turn them into the expected format for https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party
3. We then comes back to GitHub to delete the install instructions to avoid maintaining two copies for the long term after https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/{APP_NAME} is published.